### ISSUE TITLE
Deep Residual Learning for Image Recognition

### ISSUE THREAD

**Issue by traversc** [greenelab/deep-review#145]
https://arxiv.org/abs/1512.03385

> Deeper neural networks are more difficult to train. We present a residual learning framework to ease the training of networks that are substantially deeper than those used previously. We explicitly reformulate the layers as learning residual functions with reference to the layer inputs, instead of learning unreferenced functions. We provide comprehensive empirical evidence showing that these residual networks are easier to optimize, and can gain accuracy from considerably increased depth. On the ImageNet dataset we evaluate residual nets with a depth of up to 152 layers---8x deeper than VGG nets but still having lower complexity. An ensemble of these residual nets achieves 3.57% error on the ImageNet test set. This result won the 1st place on the ILSVRC 2015 classification task. We also present analysis on CIFAR-10 with 100 and 1000 layers. 

Although this paper isn't directly related to bioinformatics, I think it is significant because it won the ImageNet competition in 2015.  The authors implement a new idea of "residual layers" to improve accuracy and drastically improve training computational time in a deep (152 layers!) CNN.  

The basic idea is to add an identity function or a projection (equivalent to a linear regression) from one layer to a subsequent layer, so that the original network essentially only computes the non-linear components of the model.  

I tried this approach out on some toy datasets using shallow neural networks, and observed similar performance to the original neural network, but much faster convergence time.

**Comment by agitter** [comment]
This is the idea that is adopted in #91, right?

**Comment by traversc** [comment]
Yes, it looks like it!
